flake.lock

```diff
@@ -20,11 +20,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1708751719,
-        "narHash": "sha256-0uWOKSpXJXmXswOvDM5Vk3blB74apFB6rNGWV5IjoN0=",
+        "lastModified": 1708976803,
+        "narHash": "sha256-yvRygcySjjSvj5JTaCdo7lPqJ/2mBV2XQ94Oaq/14qw=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "f63ce824cd2f036216eb5f637dfef31e1a03ee89",
+        "rev": "548a86b335d7ecd8b57ec617781f5e652ab0c38e",
```

Comment:
This is currently the latest commit of `nixpkgs-unstable` which breaks the other changes in this PR as discussed here https://github.com/fufexan/nix-gaming/issues/161#issuecomment-1966575062. Not sure what is the best solution, maybe update the flake.lock to an older nixpkgs commit so that it does not break by default for people who don't follow nixos-unstable?

Reply:
I think I'll wait for the fix to be merged to nixos-unstable before having thiss merged. It doesn't really resolve the issue, so it should not try to.